Bugzilla/Constants.pm

```diff
@@ -506,8 +506,8 @@ use constant INSTALLATION_MODE_NON_INTERACTIVE => 1;
 
 # Data about what we require for different databases.
 use constant DB_MODULE => {
-    # MySQL 5.0.15 was the first production 5.0.x release.
-    'mysql' => {db => 'Bugzilla::DB::Mysql', db_version => '5.0.15',
+    # Require MySQL 5.6.x for innodb's fulltext support
+    'mysql' => {db => 'Bugzilla::DB::Mysql', db_version => '5.6.12',
```

Comment:
Ooops, this actually tripped me up here:
https://bugzilla.mozilla.org/show_bug.cgi?id=1467006
https://github.com/docker-library/mariadb/issues/174